packages/dumb/src/components/tooltips/warm-up-tooltip/warm-up-tooltip.tsx

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2023, Diabeloop
+ *
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+import moment from 'moment-timezone'
+import { Tooltip } from '../../../index'
+import React, { FC } from 'react'
+import { TimePrefs } from 'medical-domain'
+import {
+  COMMON_TOOLTIP_SIDE,
+  COMMON_TOOLTIP_TAIL_HEIGHT,
+  COMMON_TOOLTIP_TAIL_WIDTH,
+  DEFAULT_TOOLTIP_BORDER_WIDTH,
+  DEFAULT_TOOLTIP_OFFSET,
+  DEFAULT_TOOLTIP_TAIL,
+  Position,
+  Side
+} from '../common/tooltip/tooltip'
+import { useTranslation } from 'react-i18next'
+import { getDateTitleForBaseDatum } from '../../../utils/tooltip/tooltip.util'
+import styles from '../warm-up-tooltip/warm-up-tooltip.css'
+import WarmUp from 'medical-domain/dist/src/domains/models/medical/datum/warm-up.model'
+import { getHourMinuteFormat } from '../../../utils/datetime/datetime.util'
+
+interface WarmupTooltipProps {
+  warmup: WarmUp
+  position: Position
+  side: Side
+  timePrefs: TimePrefs
+}
+
+export const WarmUpTooltip: FC<WarmupTooltipProps> = (props) => {
+  const {  warmup, position, side, timePrefs } = props
+  const { t } = useTranslation('main')
+  const endTime = moment.tz(warmup.epochEnd, warmup.timezone).format(getHourMinuteFormat())
+
+  return (
+    <Tooltip
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**🔨 issue:** ‏The `borderColor` property is missing here, leading to the tooltip border being black when it should be grey:

On this branch:
<img width="217" alt="image" src="https://github.com/mdblp/blip/assets/116577202/0072120c-d29b-406f-94bd-9028e7834770">

On main:
<img width="216" alt="image" src="https://github.com/mdblp/blip/assets/116577202/2cf2541d-3035-4677-9440-12af9c7f4f93">